docs/OVTF_integration/OVTF_build.md

```diff
@@ -0,0 +1,327 @@
+.. _Prerequisites:
+@endsphinxdirective
+
+## 1. Prerequisites
+
+The first step is cloning the `openvino_tensorflow` repository:
+
+```bash
+$ git clone https://github.com/openvinotoolkit/openvino_tensorflow.git
+$ cd openvino_tensorflow
+$ git submodule init
+$ git submodule update --recursive
+```
+
+
+@sphinxdirective
+.. _Ubuntu:
+@endsphinxdirective
+
+### Ubuntu
+
+1. Install apt packages  
+
+      ```bash  
+      # users may need sudo access to install some of the dependencies  
+      $ apt-get update
+      $ apt-get install python3 python3-dev
+      $ apt-get install -y --no-install-recommends ca-certificates autoconf automake build-essential \
+      libtool unzip git unzip wget zlib1g zlib1g-dev bash-completion \
+      build-essential cmake zip golang-go locate curl clang-format cpio libtinfo-dev jq \
+      lsb-core libusb-1.0-0-dev patchelf
+    
+      # install the required gcc package (Optional; if gcc-7 is not installed)
+      $ apt-get install gcc-7 g++-7  
+
+      # if multiple gcc versions are installed then install alternatives
+      $ update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 70 
+      $ update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 70
+
+      # configure alternatives (select the required gcc version)
+      $ update-alternatives --config gcc
+      $ update-alternatives --config g++
+      ```        
+
+2. Install Python requirements  
+
+      ```bash
+      $ pip3 install -r requirements.txt
+      ```  
+
+3. Install CMake, supported version >=3.14.0 and =<3.20.0, 
+        
+      ```bash
+      # to install CMake 3.18.2
+      $ wget https://github.com/Kitware/CMake/releases/download/v3.18.4/cmake-3.18.4-Linux-x86_64.tar.gz && \
+      $ tar -xzvf cmake-3.18.4-Linux-x86_64.tar.gz && \
+      $ cp cmake-3.18.4-Linux-x86_64/bin/* /usr/local/bin/ && \
+        cp -r cmake-3.18.4-Linux-x86_64/share/cmake-3.18 /usr/local/share/
+      ```
+
+4. Install Bazelisk (Optional; required only while building TensorFlow from source)
+
+      ```bash
+      $ apt-get update && apt-get install -y openjdk-8-jdk
+      $ curl -fsSL https://deb.nodesource.com/setup_12.x | bash -
+      $ apt-get install -y nodejs
+      $ npm install -g @bazel/bazelisk
+      ```
+@sphinxdirective
+.. _macOS:
+@endsphinxdirective
+
+### macOS
+
+1. Install HomeBrew packages
+
+      ``` bash
+      $ brew install python@3.9
+      $ brew install cmake autoconf automake libtool libusb wget 
+      ```
+
+2. Install Pip and requirements
+
+      ```bash
+      $ wget https://bootstrap.pypa.io/get-pip.py
+      $ python3 get-pip.py
+      $ pip3 install -r requirements.txt
+      ```
+
+3. Install Bazelisk v1.10.1 (Optional; required only while building TensorFlow from source)
+
+      ```bash
+      $ wget https://github.com/bazelbuild/bazelisk/releases/download/v1.10.1/bazelisk-darwin-amd64
+      $ mv bazelisk-darwin-amd64 /usr/local/bin/bazel
+      $ chmod 777 /usr/local/bin/bazel
+      ```
```

Comment:
why all this stuff is a part of this document? It should be readme.md (or any similar document) of OVTF itself!